examples/va/hci/edpm-pre-ceph/values.yaml

```diff
@@ -47,6 +47,10 @@ data:
           {%- endfor %}
           {% set min_viable_mtu = mtu_list | max %}
           network_config:
+          - type: interface
+            name: nic1
+            use_dhcp: true
+            mtu: {{ min_viable_mtu }}
```

Comment:
Is this change supposed to be included here?  It's changing a VA, but this PR is adding a new DT.

Reply:
I thought it will not harm to have this. Thanks for pointing. Removing the change